atelier-assembler/src/lib.rs

```diff
@@ -435,7 +435,7 @@ impl ModelAssembler {
     }
 
     fn read_model(&self, path: &Path) -> Result<Model> {
-        info!("ModelAssembler::read_model({:?})", path);
+        trace!("ModelAssembler::read_model({:?})", path);
```

Comment:
I can see moving some debug messages to trace, but info?

Reply:
I treat `trace` as things that trace the execution path, which conventionally for me falls into the "we entered this function" or "we entered this branch".

But that's a me'ism. Glad to revert all these. It was just a way for me to reduce (and then focus on the right) the noise while debugging, so "RUST_LOG=info" got quite a bit quieter.

Reply:
The only issue I have is that trace is two levels lower, I like info for entry-point functions, I want to see the shape of the execution and have debug and trace for the details